scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/solver/PrioritySolver.java

```diff
@@ -511,29 +524,34 @@ private void satisfyGoalGeneral(Goal goal) {
     assert plan != null;
 
     //continue creating activities as long as goal wants more and we can do so
+    logger.info("Starting conflict detection before goal " + goal.getName());
     var missingConflicts = getConflicts(goal);
+    logger.info("Found "+ missingConflicts.size() +" conflicts in conflict detection");
```

Comment:
Minor, can ignore: merge related logs
```suggestion
    var missingConflicts = getConflicts(goal);
    logger.info("Found "+ missingConflicts.size() +" conflicts in conflict detection for goal "  +goal.getName());
```

Reply:
Will ignore, we want to delineate before and after conflict detection.